ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiVersionService.scala

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.apiserver.services
+
+import com.daml.dec.DirectExecutionContext
+import com.daml.ledger.api.v1.version_service.GetApiVersionRequest
+import com.daml.ledger.api.v1.version_service.GetApiVersionResponse
+import com.daml.ledger.api.v1.version_service.VersionServiceGrpc
+import com.daml.ledger.api.v1.version_service.VersionServiceGrpc.VersionService
+import com.daml.platform.api.grpc.GrpcApiService
+import io.grpc.ServerServiceDefinition
+
+import scala.concurrent.Future
+import scala.io.Source
+
+private[apiserver] final class ApiVersionService private
+    extends VersionService
+    with GrpcApiService {
+
+  private val versionFile: String = "ledger-api/VERSION"
```

Comment:
Afaict, this file won’t be there outside of our repo. I see two options:

1. Either pack it in the resources and call `rlocation` like we do for other resources. This is packed into the JAR so it will be distribued.
2. Or generate a source file based on this file and compile that.

I’d probably do 1 but in the end it doesn’t matter all that much.

Reply:
So by using `rlocation` I can avoid manually placing the version file in the resources using `filegroup`?

Reply:
I’ll take that back. You need `rlocation` to get thigns in `data`. For `resources`, this should work fine.